x-pack/plugins/file_upload/public/components/index_name_form_async_wrapper.tsx

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Should the async wrapper components live in the API folder since that is where they are used?